src/pages/Contribution.pug

```diff
@@ -11,80 +11,77 @@ block article
+        Right from the beginning, start a **list with every person** that contributed to the project in any way, like ‘experts’, ‘supervisors’, ‘sponsors’. 
         
-        - Use a calendar to keep track of every meeting happening. You can also use it for vacation periods for members of your team. 
+        Use a **calendar** to keep track of every meeting happening. It can also be used for vacation periods for members of the team. 
         
-        - You may want to use ‘github’ for your wiki.
-        It allows the wiki team to set up the wiki from github easily; it keeps the progress structured and allows constant backups in case some things go wrong.
+        **‘github’** could be used for the wiki.
+        It allows the wiki team to set up the wiki from github easily. Furthermore, it keeps the progress structured and allows constant backups in case some things go wrong.
         
         # Laboratory Subgroup
-        - It is recommendable to have at least one meeting a week. This year, we had two meetings a week: one general update meeting and one on Fridays to plan experiments for the next week. 
+        It is recommendable to have at least **one meeting a week**. This year, we had two meetings a week: one general update meeting and one on Fridays to plan experiments for the next week. 
 
-        - You can plan available people for the next week or next weeks by implementing a shift availability schedule into Teams. The subgroup leader then used this shift availability schedule to plan the shifts for the next week in a Google Calendar. We used a simple Excel table:
+        The **lab shifts** can be scheduled in the next week with available persons by implementing a shift availability schedule into Teams. The Subgroup Leader then used this shift availability schedule to plan the shifts for the next week in a Google Calendar. We used a simple Excel table:
 
     - var imgpath = "" + require("../assets/img/contribution/Schedule.jpg")["default"]
     +image(1, imgpath, "Schedule Planner for Laboratory Subgroup")
 
     :markdown-it(html)
-        - It is advisable to assign specific tasks to group members, e.g. one or two people responsible for any orders, people who focus more on research or communication with supervisors and experts. 
+        It is advisable to assign **specific tasks to group members**, e.g. one or two people responsible for any orders, people who focus more on research or communication with supervisors and experts. 
 
-        - Additionally to the shift availability schedule, we implemented a general availability plan for the remaining weeks of the competition in order to estimate the available capacities. Example of how this can look like:
+        Additionally to the shift availability schedule, we implemented a **general availability plan** for the remaining weeks of the competition in order to estimate the available capacities. For example, this may look like this:
```

Comment:
```suggestion
        Additionally to the shift availability schedule, we implemented a **general availability plan** for the remaining weeks of the competition in order to estimate the available capacities. For example, it could look like this:
```